ui/Screen/Org.svelte

```diff
@@ -30,14 +30,15 @@
 
   import ProjectsMenu from "./Org/ProjectsMenu.svelte";
   import MembersMenu from "./Org/MembersMenu.svelte";
+  import MenuItem from "../DesignSystem/Component/HorizontalMenu/MenuItem.svelte";
 
   const menuRoutes = {
     "/orgs/:id/": ProjectsMenu,
     "/orgs/:id/projects": ProjectsMenu,
     "/orgs/:id/members": MembersMenu,
   };
 
-  const topbarMenuItems = orgId => [
+  const topbarMenuItems = (orgId: string) => [
```

Comment:
Should `orgId` be a type `OrgId` to disambiguate it from other `string`s?

Reply:
ja i think that would make sense for all of the `id`s

Reply:
Sehr gut